src/engine/layout.jl

```diff
@@ -172,7 +172,21 @@ function tex_layout(expr, state)
                     (rightinkbound(content), 0)
                 ]
             )
+        elseif head == :overline
```

Comment:
Can you move this block so that it is in alphabetical order (i.e. between `:function` and `:space`) ? It is quite minor, but I think it helps down the line.

Reply:
Done.